### PR TITLE
Merchant now greets the player!

### PR DESCRIPTION
**Pull request by Schtekt**
This pull request is based of the code of the new shop PR, do not review code from that, only the sounds themselves and the method of playing them.

EDIT:
Also added the player walk sound to this PR as it was so little code :)

### CODE REVIEW

File: StortSpelProjekt/Game/src/main.cpp [hunk 1]
```diff
@@ -700,6 +700,12 @@ Scene* ShopScene(SceneManager* sm)
     TextureCubeMap* skyboxCubemap = al->LoadTextureCubeMap(L"../Vendor/Resources/Textures/CubeMaps/skymap.dds");
 
 	AudioBuffer* music = al->LoadAudio(L"../Vendor/Resources/Audio/shopMusic.wav", L"ShopMusic");
+    AudioBuffer* gawblinSound = al->LoadAudio(L"../Vendor/Resources/Audio/IgnoredAudio/HelloThere.wav",L"HelloThere");
```
Comment (LeoWikstrom) [inline StortSpelProjekt/Game/src/main.cpp]: Doesn't work, wrong path

**Review by LeoWikstrom — CHANGES_REQUESTED** [review]

File: StortSpelProjekt/Game/src/main.cpp (hunk 1 above)
Comment (LeoWikstrom) [reply StortSpelProjekt/Game/src/main.cpp]: Also, sounds need to be louder. They sound good, but can barely be heard

**Review by LeoWikstrom — DISMISSED** [review]
Love the voice 👍🏽

**Review by fredrix96 — CHANGES_REQUESTED** [review]
I love the added sounds, but I still think that the volume is too low. However, the walk sound seems to sometimes suddenly stop playing after fighting or jumping. It also seems to be playing during jumps

**Review by LeoWikstrom — APPROVED** [review]

**Review by fredrix96 — APPROVED** [review]
Seems fine now!

**Pull request merged**